src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c

```diff
@@ -835,6 +835,8 @@ printUsage(void)
  "transport=<name>                 transport spec                    none\n"
  "address=<listen/attach address>  transport spec                    \"\"\n"
  "server=y|n                       listen for debugger?              n\n"
+ "allow=<IP|IP-list>               If server=y, allows connections only from the IP addresses/subnets specified.\n"
+ "                                                                   * (allows connection from any address)\n"
```

Comment:
I'm just wondering if we should also mention to use "+" to separate addresses in the list.

Reply:
Yes we can add this, I posted another commit adding some information.